packages/default/scss/upload/_layout.scss

```diff
@@ -27,7 +27,7 @@
             border-width: 0;
             background-color: transparent;
 
-            em {
+            .k-dropzone-hint {
```

Comment:
I think this change is for the better, however it will require a change in the rendering of the Kendo UI for jQuery upload, too. This in turn will require the change to all LESS themes -- @joneff what do you think?

Reply:
By the way, a way to keep the theme backwards-compatible is to keep both selectors for a time, and open an issue in KUI jQuery to change the rendering. Keeping both selectors increases the style size, but prevents the theme from breaking until the necessary changes are implemented across suites. Look for `BACKCOMPAT` in the themes' source for instances of this method.